app/assets/javascripts/pdfjs_viewer/viewer.js.erb

```diff
@@ -7141,12 +7141,11 @@ var PDFViewerApplication = {
 };
 window.PDFView = PDFViewerApplication; // obsolete name, using it as an alias
 
+var HOSTED_VIEWER_ORIGINS = <%= ENV.fetch("PDFJS_VIEWER_ORIGINS", "").split(",").inspect %>;
 
-var HOSTED_VIEWER_ORIGINS = ['null',
-  'http://mozilla.github.io', 'https://mozilla.github.io'];
 function validateFileURL(file) {
   try {
-    var viewerOrigin = new URL(window.location.href).origin || 'null';
+    var viewerOrigin = window.location.origin;
```

Comment:
I thought that this does the same thing (it's more readable). But according to https://developer.mozilla.org/en-US/docs/Web/API/Window/location you can't use `window.location.origin` is only available since IE 11, I think this is an IE workaround.
See this: http://tosbourn.com/a-fix-for-window-location-origin-in-internet-explorer/

I'll undo that change.